modules/apps/sharing/sharing-document-library/src/main/java/com/liferay/sharing/document/library/internal/display/context/SharingDLViewFileVersionDisplayContext.java

```diff
@@ -138,18 +138,46 @@ else if (Objects.equals(
 			i++;
 		}
 
+		DropdownItem sharingDropdownItem =
```

Comment:
Suggestion:

To buy space and to make easier the deletion of the FF, make this instead this code

```
if (FeatureFlagManagerUtil.isEnabled("LPS-197477")) {
			UnsafeConsumer<DropdownContextItem, Exception> sharingActionItem =
				_sharingDropdownItemFactory.createShareActionUnsafeConsumer(
					DLFileEntryConstants.getClassName(),
					_fileEntry.getFileEntryId(), _httpServletRequest);

			if (i >= dropdownItems.size()) {
				dropdownItems.addAll(
					DropdownItemListBuilder.addContext(
						sharingActionItem
					).build());
			}
			else {
				dropdownItems.addAll(
					i,
					DropdownItemListBuilder.addContext(
						sharingActionItem
					).build());
			}
		}
		else {
			DropdownItem sharingDropdownItem =
				_sharingDropdownItemFactory.createShareDropdownItem(
					DLFileEntryConstants.getClassName(),
					_fileEntry.getFileEntryId(), _httpServletRequest);

			if (i >= dropdownItems.size()) {
				dropdownItems.add(sharingDropdownItem);
			}
			else {
				dropdownItems.add(i, sharingDropdownItem);
			}
		}
```

Reply:
Done!